developerSandboxSetup.md

```diff
@@ -31,100 +31,174 @@ There are three options for running a sandbox.
 
-The install directions are located [here](https://www.factom.com/howto/)
+Directions for installing Factom Federation may be found [here](https://docs.factom.com/#install-factom-federation-ff)
 
-Here are the binaries for Windows, Mac, and Linux: https://github.com/FactomProject/distribution
+Installers for for Windows, Mac, and Linux are hosted at https://github.com/FactomProject/distribution
 
 Install the binaries like you would any other for your OS. The install directions walk you through various operating systems.  Do not run them yet, as you will be making your own fresh blockchain instead of using the public one. If you did run factomd before setup, follow the directions below for resetting the blockchain.
 
-Three programs are installed: factomd, fctwallet, and factom-cli.
+Three programs are installed: factomd, factom-walletd, and factom-cli. You may also wish to install the GUI [Enterprise Wallet](https://docs.factom.com/#enterprise-wallet).
 
 * Factomd is the main program.  It manages the blockchain, connects to the public network, and enforces the network rules.
-* Fctwallet is an application for holding private keys. It builds Factoid transactions and handles crypto related operations to add user data into Factom.
-* Factom-cli is a program for users to interface with factomd and fctwallet. Through this they create Chains, Entries, and Factoid transactions.
+* factom-walletd is an application for holding private keys. It builds Factoid transactions and handles crypto related operations to add user data into Factom.
+* Factom-cli is a program for users to interface with factomd and factom-walletd. It may be used to create Chains, Entries, and Factoid transactions.
 
 #### Configure Factomd for Sandbox use
 
+Create a folder in your user home folder. The folder should be called `.factom`. If factomd has been run on your computer before this folder will already exist and may already contain part of the factom blockchain.
 
-Create a folder in your user home folder. The folder should be called `.factom`. If factomd has been run on your computer before you may need to first rename the database that is already there: `mv ~/.factom/ldb ~/.factom/ldb_old`.
+In a terminal Linux and Mac: `mkdir -p ~/.factom/m2`or Windows: `mkdir %HOMEDRIVE%%HOMEPATH%\.factom\m2`
 
-In a terminal Linux and Mac: `mkdir ~/.factom`or Windows: `mkdir %HOMEDRIVE%%HOMEPATH%\.factom`
-
-Save the configuration file [factomd.conf](https://raw.githubusercontent.com/FactomProject/FactomCode/master/factomd/factomd.conf) to your `.factom` directory.
+Save the configuration file [factomd.conf](https://raw.githubusercontent.com/FactomProject/factomd/master/factomd.conf) to your `.factom/m2` directory.
 
 * Open `factomd.conf`
+* Change the line `Network` from `MAIN` to `LOCAL`
+	* This will cause factomd to run on its own local network rather than the factom main network.
 * Change the line `NodeMode` from `FULL` to `SERVER`
```

Comment:
nodemode doesn't do anything anymore.  all that matters is the identity and the prefix.  if there is no prefix, it considers itself a leader.